src/oemof/thermal/heat_pump.py

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+import traceback
+
+def get_cop(temperature, heatpump_type = "Air", water_temp = 60):
+    """ Calculation of the coefficient of performance depending
+    on the outside temperature
+    
+    Parameters
+    ----------
+    temperature: time series
+        countaining the outside temperature
+    heatpump_type: string
+        defines the technology used. Ground is more efficient than Air.
+    water_temp: int
+        temperature needed for the heating system
+        
+    References
+    ----------
+    ..  [1]: 'https://www.researchgate.net/publication/255759857_A_review_of_domestic_heat_pumps'
+        Research paper about domestic heatpumps, containing the formulas used
+    """
+
+    cop_lst = []
+    
+    if heatpump_type == "Air":
+        for tmp in temperature:
+            cop = (6.81 - 0.121 * (water_temp - tmp)
+                   + 0.00063 * (water_temp - tmp)**2)
+            cop_lst.append(cop)
+    
+    elif heatpump_type == "Ground":
+        for tmp in temperature:
+            cop = (8.77 - 0.15 * (water_temp - tmp)
+                   + 0.000734 * (water_temp - tmp)**2)
+            cop_lst.append(cop)
+    
+    else:
+        traceback.print_exc("Heatpump type is not defined")
+        return None
+
```

I do not want to be too petty and I think we can merge it without changing it but just for the protocol:

If an invalid parameter is given that leads to an invalid result you should raise an exception. If you return `None` the script will go on but we already know that it will lead to an exception later on. The user will have to scroll up to find the phrase "Heatpump type is not defined" between all the information given. As we know that `None` is not a valid COP we can raise a exception right away and the user can fix it directly. 
```diff
else:
-      traceback.print_exc("Heatpump type is not defined")
-      return None
+      msg = "'{0}' is not a valid heatpump type. Use 'Air' or 'Ground'."
+      raise ValueError(msg.format(heatpump_type)) 
```
The reason for such an error is most of the time just a typo. The user will get something like:
```
Value Error: 'AiR' is not a valid heatpump type. Use 'Air' or 'Ground'
```